packages/litexa-html/README.md

```diff
@@ -44,7 +44,119 @@ project_dir
         └── html
 ```
 
-## Adding 'mark' SSML Tags
+## Alexa Web API Interface
+
+Integrating your skill with the Alexa Web API will allow it to both send and receive data
+from your web app.
+
+### Skill Manifest
+
+Before your skill can send and receive data through the Alexa Web API, you will need to
+specify the interface in your skill's manifest. You will need to include
+`ALEXA_PRESENTATION_HTML` in your `skill.*` file to let Alexa know that you are using it.
+For example, your skill manifest might look something like this:
+
+```json
+{
+  "manifest": {
+    "apis": {
+      "custom": {
+        "interfaces": [
+          {
+            "type": "ALEXA_PRESENTATION_HTML"
+          }
+        ]
+  ...
+}
+```
+
+### Directives
+
+With Alexa Web API directives, you can send data to your web app from within your skill.
+
+Use the `directive` keyword in your `*.litexa` files to add these directives to your
+Alexa responses. You can get more information about the usage of the `directive` keyword
+from our
+[Litexa language reference](https://litexa.com/reference/#directive).
+
+#### Start Directive
+
+To start your web app, attach the `Alexa.Presentation.HTML.Start` directive to your skill's response.
+Here's an example of what your Litexa code could look like:
+
+```coffeescript
+launch
+    if HTML.isPresent()
+      directive htmlStartDirective(webAppUrl)
+    ...
+```
+
+And the `#htmlStartDirective()` function would return an object like this, and Litexa will add
+it to your skill's response:
+
+```javascript
+function htmlStartDirective(url) {
+    return {
+        type: "Alexa.Presentation.HTML.Start",
+        configuration: {
+            timeoutInSeconds: 1000
+        },
+        data: {},
+        request: {
+            uri: url,
+            method: "GET",
+            headers: {}
+        },
+        transformers: {}
```

Comment:
Would leave code comments to briefly explain the individual properties (timeoutInSeconds, data, transformers).